phpunit/class-gutenberg-rest-block-patterns-controller-test.php

```diff
@@ -1,221 +0,0 @@
-<?php
-/**
- * Unit tests covering Gutenberg_REST_Block_Patterns_Controller functionality.
- *
- * @package WordPress
- * @subpackage REST_API
- */
-
-/**
- * Unit tests for REST API for Block Patterns.
- *
- * @group restapi
- * @covers Gutenberg_REST_Block_Patterns_Controller_6_2
- */
-class Gutenberg_REST_Block_Patterns_Controller_6_2_Test extends WP_Test_REST_Controller_Testcase {
```

Comment:
This function is not in Gutenberg anymore. cc @ntsekouras  for confirmation. Let's also confirm that the tests were backported properly.

Reply:
Yep, here: https://github.com/WordPress/wordpress-develop/pull/3892